Documentation/Settings.cfg

```diff
@@ -1,55 +1,66 @@
+# More information about this file:
+# https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/FileStructure.html#settings-cfg
+
 [general]
 
 project     = TYPO3 Explained
 version     = 11.5
 release     = 11.5
-t3author    = Documentation Team
-copyright   = since 2000 by the TYPO3 Documentation Team
-description = Main TYPO3 Core documentation
+copyright   = since 2012 by the TYPO3 contributors
 
 [html_theme_options]
 
-github_branch        = 11.5
-github_commit_hash   =
+# "Edit on GitHub" button
 github_repository    = TYPO3-Documentation/TYPO3CMS-Reference-CoreApi
-github_revision_msg  =
-github_sphinx_locale =
-project_contact      = mailto:documentation@typo3.org
-project_discussions  =
-project_home         =
-project_issues       = https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues
+github_branch        = 11.5
+
+# Footer links
+project_home         = https://docs.typo3.org/m/typo3/reference-coreapi/11.5/en-us/
+project_contact      = https://typo3.slack.com/archives/C028JEPJL
 project_repository   = https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi
-use_opensearch       = https://docs.typo3.org/m/typo3/reference-coreapi/11.5/en-us/
+project_issues       = https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues
+project_discussions  =
 
+use_opensearch       =
 
 [intersphinx_mapping]
 
-# official TYPO3 manuals
-h2document    = https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/
-t3contribute  = https://docs.typo3.org/m/typo3/guide-contributionworkflow/main/en-us/
-t3editors     = https://docs.typo3.org/m/typo3/tutorial-editors/11.5/en-us/
-t3extbasebook = https://docs.typo3.org/m/typo3/book-extbasefluid/11.5/en-us/
-t3install     = https://docs.typo3.org/m/typo3/guide-installation/11.5/en-us/
-t3sitepackage = https://docs.typo3.org/m/typo3/tutorial-sitepackage/11.5/en-us/
-t3start       = https://docs.typo3.org/m/typo3/tutorial-getting-started/11.5/en-us/
-t3tailor      = https://docs.typo3.org/other/typo3/tailor/main/en-us/
-t3tca         = https://docs.typo3.org/m/typo3/reference-tca/11.5/en-us/
-t3tsconfig    = https://docs.typo3.org/m/typo3/reference-tsconfig/11.5/en-us/
-t3tsref       = https://docs.typo3.org/m/typo3/reference-typoscript/11.5/en-us/
-t3ts45        = https://docs.typo3.org/m/typo3/tutorial-typoscript-in-45-minutes/11.5/en-us/
-t3viewhelper  = https://docs.typo3.org/other/typo3/view-helper-reference/11.5/en-us/
-
+# Official TYPO3 manuals
+h2document     = https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/
+# t3cheatsheets  = https://docs.typo3.org/m/typo3/docs-cheatsheets/main/en-us/
```

Comment:
I am wondering if we should also link all the commented out versions to 11.5 if available. Such that if they are needed ppl don't accidently link to main because they just comment it in? What do you think?

Reply:
Mh, i would not like to switch now in the middle. My current rule is making them all always `main` if not in use. In the next run you could switch that, but i am not sure if it is worth the effort.